chrome_extension/cardboard/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "manifest_version": 3,
+  "name": "Cardboard Chrome Extension",
+  "version": "1.0",
+  "icons": {

```

Comment:
If I recall correctly you can just specify one of these if it's the same image (whatever the actual size is) and chrome scales it automatically

Reply:
I tried one version of this and it continued to show for the Extension itself, but didn't show in the Extensions "store". I can play around with this more in another PR.